superset-frontend/src/features/databases/DatabaseModal/SqlAlchemyForm.tsx

```diff
@@ -38,17 +38,13 @@ const SqlAlchemyTab = ({
   testInProgress?: boolean;
   children?: ReactNode;
 }) => {
-  let fallbackDocsUrl;
-  let fallbackDisplayText;
-  if (SupersetText) {
```

Comment:
`SupersetText` is either a module or `{}`, and both a truthy.

https://github.com/apache/superset/blob/ec9e9a46f2f092ce56d3ed5a8a9a3ea0214db88a/superset-frontend/src/utils/textUtils.ts#L18-L25